docs/content/patterns/alz/Update-to-new-Release.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Update to a new release
+geekdocCollapseSection: true
+weight: 71
+---
+
+## What is the latest release
+
+The list of enhancement, additions and fixed bugs contained in every release can be seen by navigating to corresponding page linked in the home page of the [azure-monitor-baseline-alerts](https://github.com/Azure/azure-monitor-baseline-alerts) repository.
+
+On the center-left side of the page there's a specific section indicating the latest release. Click on the release number to see the release content.
+
+![Releases](../media/Releases.png)
+
+</br>
+
+## Steps to update to the latest release
+
+This section will guide you through the necessary steps to update the current deployment with the latest enhancements contained in the latest release. Guidance on updating to releases containing breaking changes is not in scope and could eventually be described in a dedicated pages. As an high-level overview you should:
+
+1. Sync your fork
+2. Update your local copy of the repo
+3. Deploy
+
+### Sync your fork
+
+Synching a fork means that we're making sure any update made to the main branch in the official repo is replicated over yours. To sync your fork:
+
+1. Open the your GitHub repo page. If you don't remember the URL, consider that it normally looks like `https://github.com/***<YourGitHubId>***/azure-monitor-baseline-alerts`.
+2. You should be redirected to the default page which is **<> Code**. If not, click on the **<> Code** tab.
+
+    ![<> Code](../media/GitHub_Code.png)
+
+3. Click on the **Sync fork** and then select **Update branch**
+
+    ![Update branch](../media/UpdateBranch.png)
+
+4. Refresh the page and make sure to see the description reported in the picture below on the left of the **Contribute** and **Sync fork** buttons
+
+    ![Branch is up to date](../media/BranchUpToDate.png)
+
+</br>
+
+### Update your local copy
+
+Within the code editor of your choice, make sure you pull the changes from your remote fork. Pulling changes from origin/main is necessary to ensure that the local branch is exactly the same as the remote. In this guide, we will use VSCode.
+
+1. Open VSCode and open the folder containing the cloned repo.
+2. In the bottom-left corner click on the pull icon
+
+    ![Pull icon](../media/PullIcon.png)
+
+3. Confirm or approve the operation if necessary
+
+    ![Confirm pull](../media/ConfirmPull.png)
+
+4. Ensure there are no issues with pulling changes and that the numbers close to the icon in the bottom-left corner are both **0** or not showing at all
+
+    ![Pull request completed](../media/PullCompleted.png)
+
+</br>
+
+### Deploy
```

Comment:
What about if you deploy using non-GH approach?

Reply:
done